manifests/server/config.pp

```diff
@@ -208,13 +211,13 @@
       mode    => '0755',
       owner   => $user,
       group   => $group,
-      require => File[$config]
+      require => File[$config],
     }
 
     if $pidfilepath {
       file { $pidfilepath:
         ensure => file,
-        mode   => '0644',
+        mode   => $pidfilemode,
```

Comment:
if you already have something that enforces the mode, why not just leave this out completely?
in that case, puppet should just use the umask.